src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -4,17 +4,22 @@
 
 package frc.robot.commands;
 
+import frc.robot.subsystems.LimelightSubsystem;
 import frc.robot.subsystems.ShooterSubsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 public class ShooterPrime extends CommandBase {
   private ShooterSubsystem m_ShooterSubsystem;
+  private LimelightSubsystem m_LimelightSubsystem;
   private int i;
 
   /** Creates a new ShooterPrimary. */
-  public ShooterPrime(ShooterSubsystem shooterSubsystem) {
+  public ShooterPrime(ShooterSubsystem shooterSubsystem, LimelightSubsystem limelightSubsystem) {
     addRequirements(shooterSubsystem);
     m_ShooterSubsystem = shooterSubsystem;
+    m_LimelightSubsystem = limelightSubsystem;
+    SmartDashboard.putBoolean("wheelReady", false);
```

Comment:
Would be better if we considered adding these to a different tab in shuffleboard.

We can determine the full UI later, but at least we will start with them on a new tab.